interfaces/apparmor/backend.go

```diff
@@ -966,9 +973,15 @@ func (b *Backend) addContent(securityTag string, snapInfo *snap.Info, cmdName st
 	}
 }
 
+// Allow optional trailing ' ' after "###PROMPT###"
+var promptReplacer = regexp.MustCompile("###PROMPT### ?")
```

Comment:
Would it make more sense to move this declaration above `prepareProfiles()` where it gets used?

Reply:
I moved it to just before `addContent`, which is where it is called from.